src/Dialog.tsx

```diff
@@ -20,7 +20,7 @@ type StyledDialogBaseProps = {
   SxProp
 
 const DialogBase = styled.div<StyledDialogBaseProps>`
-  box-shadow: 0px 4px 32px rgba(0, 0, 0, 0.35);
+  box-shadow: ${get('shadows.shadow.large')};
```

Comment:
couldn't find a primer/css file for dialog, so I'm guessing at some of these. Dark mode doesn't seem quite right:

![image](https://user-images.githubusercontent.com/16739070/109690071-8fc8d780-7b4b-11eb-889d-fd36f325ca67.png)

Reply:
Here's the Primer CSS equivalent for Dialog: https://primer-css-p02l95sun-primer.vercel.app/css/components/box-overlay

Looks like the code for that actually lives in github/github: https://github.com/github/github/blob/master/app/assets/stylesheets/components/box-overlay.scss